gnovm/cmd/gno/util.go

```diff
@@ -100,6 +100,60 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+func gnoPackagePathsFromPattern(patterns []string) ([]string, error) {
```

Comment:
what is the logic behind adding these functions on util.go instead on pkg.go?

Reply:
Just followed the existing style